securedrop/models.py

```diff
@@ -297,17 +297,21 @@ def to_api_v2(self) -> Dict[str, Any]:
         # (format: {interaction_count}-{journalist_filename}-*)
         interaction_count = int(self.filename.split("-")[0])
 
-        return {
+        d = {
```

Comment:
nit: can we avoid 1-letter variables? I assume this is short for "data", which is far easier to read :)

Reply:
`data`, `dict`, *tomato*, *tomahto*....